### DISCUSSION TITLE
Task View and alt + tab not working

### DISCUSSION THREAD

**Discussion by TheKa1ma** [valinet/ExplorerPatcher#1721] [General]
I recently updated Windows 11 Version 22H2. after updating, I'm not able to use task view button, and once you click on task view button, alt + tab also stops working till you restart explorer. i updated new explorer. and i reinstalled new latest version two times that just releases 4 days ago (22621.1992.56.1).

After using little more i find out that On-screen volume and brightness slider gone as well. Brightness and volume keys working fine but on-screen sliders doesn't shows.

**Comment by qwertycxz** [comment]
Same. After I updated the KB5028254 (with the latest version of EP), I firstly encontered the unstable thing mentioned in #1707. After I switch the "Enable console" and the "dump memory leaks" for serval times. The explorer stoped keeping crushing, but the win-tab and alt-tab is still unuseable.

**Reply by TheKa1ma** [reply]
Yeahh, I really hope they fix soon. because i really hate win 11 taskbar.

**Comment by derkuci** [comment]
Same here.  Windows build 22621.2070.

**Reply by vippium** [reply]
To fix this, follow the steps :
1. Uninstall the preview update first.
2. Uninstall EP after this and remove all reg. keys.
3. After restoring to previous update, make a restart after EP unistallation
4. Install EP on previous bild i.e. 1992
5. after symbols download notification, restart again.
6. Install preview update and restart
7. after restart, don't start anything and wait for symbols download msg for 2070 version.
8. after msg, restart ur device. and it will work fine without any issue.

But there is a issue in all preview updates, that action center (aka notifications) is overlapping win10 taskbar. So, keep in mind. Thanks

**Reply by derkuci** [reply]
Thanks for the suggestion.  I tried, but they didn't work for me.  One possible deviation was that, in step 7, upon restart, Windows automatically started Edge (although I didn't manually start anything).  Not sure if that made the difference, but after step 7 or 8, things went back to "broken".

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-us/topic/july-26-2023-kb5028254-os-build-22621-2070-preview-66161ea6-c106-43e9-bef5-8410ccbd34e8

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

**Comment by derkuci** [comment]
Thanks for the suggestion.  I unchecked "Get the latest updates as soon as available", and disabled (paused) Windows updates.  That worked for a while, until I re-enabled the updates.  It seems that even without the preview updates, Windows was trying to upgrade to 22621.2134, with KB5028948, KB5028851, and KB5029263, and broke Task View for ExplorerPatcher.

I now go back to "no updates at all" mode.  Stick with 22621.1992.

**Reply by pyrates999** [reply]
What do you mean you re-enabled the updates?  Did you turn the setting "Get the latest updates as soon as available" back on?

**Reply by derkuci** [reply]
re-enabled means "Resume updates" (because I paused the updates).

"Get the latest updates as soon as they're available" is still off.

**Reply by pyrates999** [reply]
Ok good.  It seems this is a duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Please close this one.

**Reply by qwertycxz** [reply]
We are in *Discussion*, not *Issue*. So this is not a duplicate and cannot be closed :)

**Comment by vippium** [comment]
It was a issue with Win10 Taskbar in EP... Switch to Win11 Taskbar till it gets a fix. Mention valinet if you want fix early as he is busy with his own stuffs.

**Comment by Bavarias** [comment]
Windows 11 Ent 22H2
OS Build 22621.2134
"miscellaneous security improvements to internal OS functionality."

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Please close this one.

**Comment by Poopooracoocoo** [comment]
Closing as duplicate of https://github.com/valinet/ExplorerPatcher/issues/1687 and https://github.com/valinet/ExplorerPatcher/issues/1802
